hildon-im-virtual-keyboard-layouts/Makefile.am

```diff
@@ -1,17 +1,17 @@
-SUFFIXES = .vkb .vkb
+SUFFIXES = .vkb .xml
 
```

Comment:
This should  not have been changed in the first place, please squash as @MerlijnWajer suggested and make a new PR

Reply:
yes, that was changed because i wanted to make a sorted language list, and for that i brought the xml files list to the file, and renamed all xml to vkb, without noticing that there are other xml strings in the file. that also caused me some time of suffering because i did not understand for some time why some files cannot be built.